tests/components/crownstone/test_config_flow.py

```diff
@@ -28,6 +29,35 @@
 from tests.common import MockConfigEntry
 
 
+@pytest.fixture(name="crownstone_setup")
+def crownstone_setup():
+    """Mock Crownstone entry setup."""
+    with patch(
+        "homeassistant.components.crownstone.async_setup_entry", return_value=True
+    ):
+        yield
+
+
+@pytest.fixture(name="pyserial_comports")
+def usb_comports():
+    """Mock pyserial comports."""
+    with patch(
+        "serial.tools.list_ports.comports",
+        MagicMock(return_value=[get_mocked_com_port()]),
+    ):
+        yield
```

Comment:
If we want to assert the serial call we can yield the mock here and include the fixture name as a test parameter.